addons/hr_recruitment/hr_recruitment.py

```diff
@@ -682,3 +690,26 @@ class applicant_category(osv.osv):
     _columns = {
         'name': fields.char('Name', required=True, translate=True),
     }
+
+
+class hr_employee(osv.Model):
+    _inherit = "hr.employee"
+
+    def _newly_hired_employee(self, cr, uid, ids, field_name, arg, context=None):
+        data = self.pool['hr.applicant'].read_group(cr, uid,
+            [('emp_id', 'in', ids), ('job_id.state', '=', 'recruit')],
+            ['emp_id'], ['emp_id'], context=context)
+        result = dict.fromkeys(ids, False)
+        for d in data:
+            if d['emp_id_count'] >= 1:
+                result[d['emp_id'][0]] = True
+        return result
+
+    def _search_newly_hired_employee(self, cr, uid, obj, name, args, context=None):
+        applicant_ids = self.pool['hr.applicant'].search_read(cr, uid, [('job_id.state', '=', 'recruit')], ['emp_id'], context=context)
+        hired_emp_ids = [applicant['emp_id'][0] for applicant in applicant_ids if applicant['emp_id']]
+        return [('id', 'in', hired_emp_ids)]
+
+    _columns = {
+        'newly_hired_employee': fields.function(_newly_hired_employee, fnct_search=_search_newly_hired_employee, type='boolean', string='Newly hired employees')
```

Comment:
This will required a little refactoring of hr_recruitment. I will talk with @tde-banana-odoo to find a clean solution ;) Maybe a new task will appear ...